### PR TITLE
3.1.0 release updates

### PR DESCRIPTION
**Pull request by thisisdano**
- Update references from `3.0.2` → `3.1.0`
- Add notifications for potential breaking changes

Please check the notifications and let me know if they look correct, clear, and complete. 

This should be the last merge to `develop` before the release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 [![CircleCI Build Status](https://img.shields.io/circleci/build/gh/uswds/uswds/develop?style=for-the-badge&logo=circleci)](https://circleci.com/gh/uswds/uswds/tree/develop) ![Snyk vulnerabilities](https://img.shields.io/snyk/vulnerabilities/npm/@uswds/uswds?style=for-the-badge) [![npm Version](https://img.shields.io/npm/v/@uswds/uswds?style=for-the-badge)](https://www.npmjs.com/package/uswds) [![npm Downloads](https://img.shields.io/npm/dt/@uswds/uswds?style=for-the-badge)](https://www.npmjs.com/package/uswds) [![GitHub issues](https://img.shields.io/github/issues/uswds/uswds?style=for-the-badge&logo=github)](https://github.com/uswds/uswds/issues) [![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4?style=for-the-badge)](https://github.com/prettier/prettier)
 
+:grinning_face_with_big_eyes:
```
Comment (amyleadem) [inline README.md:5]: This doesn't appear to be rendering.

Comment (thisisdano) [reply README.md:5]: Huh, where did that come from? It appears to be my change but I don't recall adding it. I definitely shouldn't be there.

Comment (thisisdano) [reply README.md:5]: That's a little unsettling

**Comment by thisisdano** [comment]
I'm going to make a new branch and start a new PR here.

**Comment by amyleadem** [comment]
> I'm going to make a new branch and start a new PR here.

:sad_face_with_big_eyes:

**Pull request closed without merging**